.github/actions/add-to-board-v1/src/run.ts

```diff
@@ -0,0 +1,81 @@
+import type { Core, Github } from './types'
+import getProjectBoardID from './getProjectBoardID'
+import addIssueToBoard from './addIssueToBoard'
+import getProjectBoardFieldList from './getProjectBoardFieldList'
+import moveIssueToColumn from './moveIssueToColumn'
+
+export default async function run(core: Core, github: Github): Promise<void> {
+  try {
+    const issueUrl = core.getInput('issue-url', { required: true })
+    const projectNumber = parseInt(core.getInput('project-number'))
+    const columnName = core.getInput('column-name')
+
+    if (isNaN(projectNumber)) {
+      core.setFailed('`project-number` must be a number')
+      return
+    }
+
+    // the issue card ID as a result of adding the issue to the project board
+    const issueCardIDs: string[] = []
+    const { owner } = github.context.repo
+    let issueUrls = issueUrl.includes(',') ? JSON.parse(issueUrl) : issueUrl
```

Comment:
This means you have to call the action like this:

```yml
steps:
  - uses: action
    with:
      issue-urls: ["https://issue1", "https://issue2"]
```

Couldn't we just call it as a list? That would match the syntax needed by create-an-issue for `assignees` and `labels`

```yml
steps:
  - uses: action
    with:
      issue-urls: https://issue1, https://issue2
```

Reply:
Ya I suppose we could if we want to call this on multiple issues outside of https://github.com/dequelabs/axe-api-team-public/pull/33/files#diff-c4e4b19cd19593039a8dc0cf88eb05bb5cca97c29a8a3301e42ab297d267163cR157, I can make that change